app/views/bigbluebutton_api/get_recordings.xml.builder

```diff
@@ -54,6 +54,6 @@ xml.response do
   end
   if @recordings.empty?
     xml.messageKey 'noRecordings'
-    xml.message 'There are not recordings for the meetings'
+    xml.message 'There are no recordings for the meetings'
```

Comment:
This wording is intentional, to match the response produced by BigBlueButton. Please leave it unchanged.